action-allowedlist/approved.json

```diff
@@ -0,0 +1,55 @@
+[
+    {
+      "actionLink": "actions/checkout",
+      "actionVersion": "ec3a7ce113134d7a93b817d10a8272cb61118579"
+    },
+    {
+      "actionLink": "actions/checkout",
+      "actionVersion": "a12a3943b4bdde767164f792f33f40b04645d846"
+    },
+    {
+      "actionLink": "actions/cache",
+      "actionVersion": "c64c572235d810460d0d6876e9c705ad5002b353"
+    },
+    {
+      "actionLink": "actions/create-release",
+      "actionVersion": "0cb9c9b65d5d1901c1f53e5e66eaf4afd303e70e"
+    },
+    {
+      "actionLink": "actions/setup-dotnet",
+      "actionVersion": "499789684c9a0d41c9b3f0d66a785ba17b1d51ab"
+    },
+    {
+      "actionLink": "actions/checkout",
+      "actionVersion": "5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f"
+    },
+    {
+      "actionLink": "actions/setup-python",
+      "actionVersion": "3105fb18c05ddd93efea5f9e0bef7a03a6e9e7df"
+    },
+    {
+      "actionLink": "Gr1N/setup-poetry",
+      "actionVersion": "462ac83c852d49e282a1233c4c24c5411696e7c7"
+    },
+    {
+      "actionLink": "actions/upload-artifact",
+      "actionVersion": "e448a9b857ee2131e752b06002bf0e093c65e571"
+    },
+    {
+      "actionLink": "github/super-linter",
+      "actionVersion": "286abe2b0349da9c074c0fed8e8ec0a86cd13279"
+    },
+    {
+      "actionLink": "actions/checkout",
+      "actionVersion": "5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f"
+    },
+    {
+      "actionLink": "actions/setup-dotnet",
+      "actionVersion": "a71d1eb2c86af85faa8c772c03fb365e377e45ea"
+    },
+    {
+      "actionLink": "./action-whitelist",
+      "actionVersion": null
```

Comment:
I just realized that we have a permanent chicken-and-egg problem here: as soon as you put a valid commit hash in here, you will have a new commit with a new hash!

So this is the one thing where we'll need to allow a non hash. In that case, to keep it easy, let's just use a version tag of "latest", so that everyone's use of this action always grabs the most recently published thing.

For the moment, creating a tag of "latest" can be a manual step. At some point we'll come back around and automate that: create a tag of "latest" on merge to main from a closed pull request. Don't need to solve that right here.

Oh, and the actionLink text needs to have "allowedlist" instead of "whitelist".

Reply:
updated the approved list with the latest tag, and also fixed the name for the above action.